test/multiple_ranges_test.cc

```diff
@@ -40,6 +40,8 @@ class MultipleRangesFixture : public ::benchmark::Fixture {
   // NOTE: This is not TearDown as we want to check after _all_ runs are
   // complete.
   virtual ~MultipleRangesFixture() {
+     // FIXME Should the 'if' below be checking for equality
+     // of the values in the containers, rather than their size?
```

Comment:
it should probably do both. The assert is no longer necessary as we're using for range loops (we used to loop using the index and it would crash) and instead we should be looping over both and comparing, then asserting/aborting with some sort of error.